shared/app.js

```diff
@@ -70,6 +73,8 @@ module.exports = Backbone.Model.extend({
      * Initialize the `ClientRouter` on the client-side.
      */
     if (!isServer) {
+      var ClientRouter = this.options.ClientRouter || require(defaultRouterModule);
```

Comment:
The example doesn't work with `1.0.1` because we need changes from those two PR:
https://github.com/rendrjs/rendr/pull/448
https://github.com/rendrjs/rendr/pull/450